Outlines/Bonding Curves.md

```diff
@@ -0,0 +1,15 @@
+Bonding Curves
+- Bonding Curves are pricing curves that mediate the supply of a token. (include longer definition & elaboration)
+
+- Bonding curves play a crucial role in token economics for a given crypto ecosystem. (include longer elaboration)
+
+- Provide some examples of dApps, Cryptocollectibles, FOMO3D, and other things that feature bonding curves.
+
+- Discussing the consequences of different growth rates, convexity, and functions
+     - exponential
+     - linear
+     - log
+- Making a comparison to Bitcoin’s fixed supply and how that is achieved.
+- Discuss incentive alignment for new participants and previous entrants
+- Potential integration with other governance mechanisms, market influence, and community levers of lnfluence
+- Exploring some sample solidity smart contracts that allow for some bonding curves to be fully automated.
```

Comment:
@calchulus And feel free to draw this examples from live projects if they make them available.

Reply:
yes